shared/airgap.go

```diff
@@ -8,18 +8,27 @@ import (
 	"github.com/rancher/distros-test-framework/pkg/customflag"
 )
 
-// SetupPrivateRegistry sets bastion node as private registry.
-func SetupPrivateRegistry(cluster *Cluster, flags *customflag.FlagConfig) error {
+// SetupAirgapRegistry sets bastion node for airgap registry.
```

Comment:
i know we discussed at the time, but sorry...i cant remember.

why do we decided to create a airgap file here instead of letting it all on the `/pkg/airgap.go` ?

since its only going to be used there right

Reply:
cause what u did know its perfect like

having a airgap.go with the `TestBuildAirgapCluster` and aux functions together and a separated file for private and default

Reply:
> i know we discussed at the time, but sorry...i cant remember.
> 
> why do we decided to create a airgap file here instead of letting it all on the `/pkg/airgap.go` ?
> 
> since its only going to be used there right

No worries, I can explain:
- `pkg/testcase/airgap.go` holds the shared **test** func that can be used by other airgap tests like `privateregistry` or `systemdefaultregistry`
- `shared/airgap.go` holds the supported func used by `pkg/testcase/airgap.go` and those func are not tests but more like func needed for the test...

I am thinking should I move `shared/airgap.go` to `support/airgap.go` to live closer to the test but then I was thinking that `shared/airgap.go` uses a lot of stuff from files in shared dir... so trying to decide that

Reply:
ok tks